plugins/woocommerce/tests/e2e-pw/bin/woocommerce-cleanup.php

```diff
@@ -0,0 +1,181 @@
+		if (!has_shortcode($page->post_content, 'woocommerce') && !in_array($page->post_title, $ignore_titles)) {
+			wp_delete_post($page->ID, true);
+		}
+
+	// Set the active theme to Twenty Twenty-Three
+    switch_theme('twentytwentythree');
+	}
+
+	// Optionally, you can also clear WooCommerce transients
+	wc_delete_product_transients();
+
+	// Set the site title
+    update_option('blogname', 'WooCommerce Core E2E Test Suite');
+
+	// Set the WooCommerce default country/state
+    update_option('woocommerce_default_country', 'US:CA');
+
+	// Set the WooCommerce selling locations to "Sell to all countries"
+    update_option('woocommerce_allowed_countries', 'all');
+
+    // Set the WooCommerce shipping locations to "Ship to all countries you sell to"
+    update_option('woocommerce_ship_to_countries', 'all');
+
+		// Disable taxes
+    update_option('woocommerce_calc_taxes', 'no');
+
+    // Enable coupons
+    update_option('woocommerce_enable_coupons', 'yes');
+
+    // Disable sequential coupon discounts
+    update_option('woocommerce_calc_discounts_sequentially', 'no');
+
+		// Set the WooCommerce currency to "United States (US) dollar"
+    update_option('woocommerce_currency', 'USD');
+
+		// Set the WooCommerce currency position to "left"
+    update_option('woocommerce_currency_pos', 'left');
+
+    // Set the WooCommerce thousand separator to a comma
+    update_option('woocommerce_price_thousand_sep', ',');
+
+    // Set the WooCommerce decimal separator to a period
+    update_option('woocommerce_price_decimal_sep', '.');
+
+    // Set the WooCommerce number of decimals to 2
+    update_option('woocommerce_price_num_decimals', '2');
+
+		// Delete all shipping zones
+    $shipping_zones = WC_Shipping_Zones::get_zones();
+    foreach ($shipping_zones as $zone) {
+        $zone_instance = new WC_Shipping_Zone($zone['zone_id']);
+        $zone_instance->delete();
+    }
+
+    // Delete the default shipping zone
+    $default_zone = new WC_Shipping_Zone(0);
+    $default_zone->delete();
+
+		// Delete all shipping classes
+    $shipping_classes = get_terms('product_shipping_class', array('hide_empty' => false));
+    foreach ($shipping_classes as $shipping_class) {
+        wp_delete_term($shipping_class->term_id, 'product_shipping_class');
+    }
+
+		// Disable all payment methods
+    $payment_gateways = WC()->payment_gateways->payment_gateways();
+    foreach ($payment_gateways as $gateway) {
+        update_option('woocommerce_' . $gateway->id . '_settings', array_merge(get_option('woocommerce_' . $gateway->id . '_settings', array()), array('enabled' => 'no')));
+    }
+
+		// Enable guest checkout
+    update_option('woocommerce_enable_guest_checkout', 'yes');
+
+		// Set the WooCommerce "From" email name
+    update_option('woocommerce_email_from_name', 'woocommerce');
+
+    // Set the WooCommerce "From" email address
+    update_option('woocommerce_email_from_address', 'wordpress@example.com');
+}
+
```

Comment:
Other settings I missed?

Reply:
Good job on this PR. I haven't tested it yet, but it looks promising.
Can this also remove the `wc_order_stats` table so that analytics data is empty?

Reply:
Can't remember now but will check it later, the `wc_customer_lookup` table might also require cleanup for some test.

Reply:
I truncated both tables with the latest version